app/controllers/quizzes_controller.rb

```diff
@@ -100,6 +106,9 @@ def for_groups
 
   def publish
     @quiz = Quiz.find(params[:id])
+    if @quiz.release_date > Date.today
+      authorize! :manage, @quiz
```

Comment:
Pentru ca ma gandeam ca in caz ca vrei sa dai release mai devreme la quiz sa poti sa o faci doar daca esti teacher